app/src/ble.c

```diff
@@ -72,10 +76,25 @@ static void security_changed(struct bt_conn *conn, bt_security_t level,
     }
 }
 
+static bool le_param_req(struct bt_conn *conn, struct bt_le_conn_param *param) {
+    static struct bt_conn_info info;
+
+    bt_conn_get_info(conn, &info);
+
+    if (info.role == BT_CONN_ROLE_MASTER && (param->interval_min != 6 || param->latency != 30)) {
```

Comment:
Why we are doing this is probably going o be none obvious to later readers of this code... Can we add a short comment here to hint what is being checked here and why?